cli/compose/loader/loader.go

```diff
@@ -444,6 +444,8 @@ func LoadVolumes(source map[string]interface{}) (map[string]types.VolumeConfig,
 			if volume.External.Name == "" {
 				volume.External.Name = name
 				volumes[name] = volume
+			} else {
+				logrus.Warnf("invalid to specify name for external volume: %s", volume.External.Name)
```

Comment:
I had made some changes here, but I think they were lost with a force push.

How about something like this: 

```go
logrus.Warnf("volume %s: volume.external.name is deprecated, please use volume.name", name)

if volume.Name != "" {
    return error.Errorf("volume %s: volume.external.name and volume.name conflict, only use volume.name", name)
}
```

I think we need to error if they are both set, since one will be ignored.